### PR TITLE
refactor(motion-planning): update motion planning script

### PR DESCRIPTION
**Pull request by ahiuchingau**
# Overview
This PR allows the motion planning script to be more interactive. You can now adjust the params in a json file `opentrons_hardware/scripts/motion_params.json` instead of editing the actual script.

Run the motion planning script by doing `pipenv run python -m opentrons_hardware.scripts.plan_motion`. To enable the debug log, add `--debug=True`. To specify a different config file, use `--params-config-path={JSON_PATH}`.

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

**Review by DerekMaggio — APPROVED** [review]
Looks pretty straightforward to me.

**Comment by DerekMaggio** [comment]
Will we be using this framework for writing unittests?

File: hardware/opentrons_hardware/scripts/plan_motion.py [hunk 1]
```diff
@@ -29,47 +34,55 @@
...
     },
 }
 
 
 def main() -> None:
     """Entry point."""
+
+    parser = argparse.ArgumentParser(description="Motion planning script.")
+    parser.add_argument(
+        "--params-file-path",
```
Comment (sfoster1) [inline hardware/opentrons_hardware/scripts/plan_motion.py:47]: maybe give this a shortflag like `-p` or even make it positional?

File: hardware/opentrons_hardware/scripts/plan_motion.py [hunk 2]
```diff
@@ -29,47 +34,55 @@
...
+        type=str,
+        required=False,
+        default=os.path.join(os.path.dirname(__file__) + "/motion_params.json"),
+        help="the parameter file path",
+    )
+    parser.add_argument(
+        "--debug",
```
Comment (sfoster1) [inline hardware/opentrons_hardware/scripts/plan_motion.py:55]: shortflag `-d` maybe?

**Review by sfoster1 — APPROVED** [review]
Looks good, could maybe rename some flags or make it dump the planned output to json or something but that's about it

**Comment by DerekMaggio** [comment]
> Will we be using this framework for writing unittests?

To clarify a bit on this, will we be using these JSON file to setup for testing? 
Basically will we create a ton of JSON files and call to them in fixtures? 

Or is this solely for debugging/manual testing purposes?

**Comment by ahiuchingau** [comment]
> > Will we be using this framework for writing unittests?
> 
> To clarify a bit on this, will we be using these JSON file to setup for testing? Basically will we create a ton of JSON files and call to them in fixtures?
> 
> Or is this solely for debugging/manual testing purposes?

Hey @DerekMaggio thanks for the clarification! This is only for manual testing purposes.

**Pull request merged**